exporter/elasticsearchexporter/internal/exphistogram/exphistogram.go

```diff
@@ -59,8 +59,16 @@ func ToTDigest(dp pmetric.ExponentialHistogramDataPoint) (counts []int64, values
 		}
 		lb := LowerBoundary(offset+i, scale)
 		ub := LowerBoundary(offset+i+1, scale)
-		counts = append(counts, int64(count))
+		counts = append(counts, safeUint64ToInt64(count))
 		values = append(values, lb+(ub-lb)/2)
 	}
 	return
 }
+
+func safeUint64ToInt64(v uint64) int64 {
+	if v > math.MaxInt64 {
+		return math.MaxInt64
```

Comment:
This results in wrong values without notifying the user.
IMO, we can't just silently go on with modified count values here.
Or it needs to documented very well.